__tests__/distributors/adoptium-installer.test.ts

```diff
@@ -0,0 +1,164 @@
+import { HttpClient } from '@actions/http-client';
```

Comment:
file name should be changed from __tests__/distributors/adoptium-installer.test.ts to __tests__/distributors/temurin-installer.test.ts